packages/app/src/cli/services/release.ts

```diff
@@ -57,16 +57,14 @@ export async function release(options: ReleaseOptions) {
     appRelease: AppReleaseSchema
   }
 
-  const variables: AppReleaseVariables = {
-    apiKey: remoteApp.apiKey,
-    appVersionId: versionDetails.id,
-  }
```

Comment:
My thought was that 2 arguments make sense:

1. The app to release a version for
2. Which version to release

But I'm OK with going back to an arguments object. It will be more complicated than before, though, as `apiKey` is now a `MinimalOrganizationApp` and `appVersionId` is now an object with 2 IDs.